### PR TITLE
Hotfix arrange module DCA palette

### PR DESCRIPTION
**Pull request by zonky2**
Hotfix arrange module dca palette

### CODE REVIEW

File: contao/dca/tl_module.php [hunk 1]
```diff
@@ -356,6 +357,12 @@
                 'includeBlankOption' => true
             ),
             'sql'              => "varchar(64) NOT NULL default ''"
-        )
+        ),
+		'guests' => array
```
Comment (richardhj) [inline contao/dca/tl_module.php]: This will overwrite the existing 'quest' field definition.

**Comment by richardhj** [comment]
Wrong tab indention ;-)
And see line comments in addition.

**Comment by zonky2** [comment]
indention fixed - thx

File: contao/dca/tl_module.php (hunk 1 above)
Comment (zonky2) [inline contao/dca/tl_module.php]: guest doesn´t have a css class per default https://github.com/contao/core/blob/master/system/modules/core/dca/tl_module.php#L741

Comment (richardhj) [inline contao/dca/tl_module.php]: Defining 'guests' with a new array will truncate the existing array and dca for 'guests'

**Comment by richardhj** [comment]
I would rather adjust MetaModel's dca instead of modifying Contao core's dca field definitions which could result into negative side effects with other third party apps.

**Comment by zonky2** [comment]
> I would rather adjust MetaModel's dca instead of modifying Contao core's dca field definitions which could result into negative side effects with other third party apps.

shure, but we have only the contao css classes to arrange the input widets...

shot348

**Comment by richardhj** [comment]
Whats about moving "Nicht zum Suchindex hinzufügen" to the end of the palette?

**Comment by zonky2** [comment]
> Whats about moving "Nicht zum Suchindex hinzufügen" to the end of the palette?

o.k. I had do this...

**Comment by discordier** [comment]
@richardhj so we are now good to go or do you have any more objections?

**Comment by richardhj** [comment]
Approved 😇

**Pull request merged**